examples/format_detection_openai_gpt.py

```diff
@@ -0,0 +1,126 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # PII detection with DiscoverX & Azure OpenAI
+# MAGIC
+# MAGIC This notebooks uses [DiscoverX](https://github.com/databrickslabs/discoverx) to run PII detection with [AZURE OpenAI API](https://learn.microsoft.com/en-us/azure/ai-services/openai/chatgpt-quickstart?tabs=command-line&pivots=programming-language-studio) over a set of tables in Unity Catalog.
+# MAGIC
+# MAGIC The notebook will:
+# MAGIC 1. Use DiscoverX to sample a set of tables from Unity Catalog and unpivot all string columns into a long format dataset
+# MAGIC 2. Run format detection with Azure OpenAI GPT model
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ## Install dependencies
+
+# COMMAND ----------
+
+# MAGIC %pip install openai
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ## Setup widgets
+
+# COMMAND ----------
+
+dbutils.widgets.text("secret_scope","discoverx","Secret Scope")
```

Comment:
NIT: are you using auto format with black? I was expecting a space after the comma.